src/jvmTest/kotlin/io/github/detekt/sarif4k/SarifMergingTest.kt

```diff
@@ -0,0 +1,31 @@
+package io.github.detekt.sarif4k
+
+import org.junit.jupiter.api.Test
+import java.io.File
+import kotlin.test.assertEquals
+
+class SarifMergingTest {
+    @Test
+    fun `merge output from sarifs of the same tool`() {
+        testMerge("/output.sarif.json", "/input_1.sarif.json", "/input_2.sarif.json")
```

Comment:
```suggestion
        testMerge("/output.sarif.json", "/input_1.sarif.json", "/input_2.sarif.json")
```
consider moving `/` into utility the method to simplify tests. (/ is an implementation detail of how to resolve a file relative to the test classpath, actual tests shouldn't need to worry about this.)